filebeat/docs/inputs/input-kafka.asciidoc

```diff
@@ -27,11 +27,10 @@ Example configuration:
 
 ----
 
-[id="{beatname_lc}-input-{type}-azure"]
-==== Configuring for Azure Event Hubs
+==== Configuration for Azure Event Hubs
```

Comment:
I meant that you should consider removing the entire section heading (not just the ID) so that the example flows after the previous one. If you want to keep the section heading, you do need to include the ID. It's not a big deal either way; it just seemed a bit inconsistent with our general approach in other topics where we show more than one example. I don't like pushing the configuration options further down the page unless there's a compelling reason (for example, if you think it's really important to call out the example, you want to link to the section, or you want a link to the section to appear under in the "On this page" set of links. Sorry for the confusion.

Reply:
oh ok, I wasn't sure if the "Configuring" -> "Configuration" was meant to be kept :-) ok I've removed the whole header